BigFraction/BigFraction.cs

```diff
@@ -191,12 +191,12 @@ public BigFraction(int i)
         //Override Equals
         public override bool Equals(object obj)
         {
-            if (obj == null) { return false; } 
+            if (obj == null) { return false; }
```

Comment:
Can you update `Equals` to properly check if object is not null and of the correct type?

```csharp
var comparebigfrac = obj as BigFraction;
if (comparebigfrac == null) { return false; }
```

Reply:
Updated.